app/main.py

```diff
@@ -1,34 +1,60 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(
+            self,
+            row: str,
+            column: str,
+            is_alive: bool = True
+    ) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: tuple,
+            end: tuple,
+            is_drowned: bool = False
+    ) -> None:
+        self.is_drowned = is_drowned
+        self.decks = self.place_ship_decks(start, end)
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+    def place_ship_decks(self, start: tuple, end: tuple) -> list:
+        ship_length = abs(sum(start) - sum(end)) + 1
+        decks = []
+        for i in range(ship_length):
+            row = start[0] + i * (start[0] < end[0])
+            col = start[1] + i * (start[1] < end[1])
+            decks.append(Deck(row=row, column=col))
```

Comment:
just do loop inside loop:
for row_index in range(specify from and to):
     for column_index in range(specify from and to):
           # append deck